AutoRest/Generators/CSharp/Azure.CSharp.Tests/AcceptanceTests.cs

```diff
@@ -685,5 +686,64 @@ public void CustomNamedRequestIdTest()
                 Assert.Equal("123", response.RequestId);
             }
         }
+        
+        [Fact]
+        public void ParameterGroupingTests()
+        {
+            const int body = 1234;
+            const string header = "header";
+            const int query = 21;
+            const string path = "path";
```

Comment:
For readibility, suggest rename the variable such as "pathParameter", "bodyParameter", etc